### PR TITLE
Update blueprint

### PR DESCRIPTION
**Pull request by jvega190**
- Make Preview Toolbar configurable - ui.xml update https://github.com/craftercms/craftercms/issues/4784
- Use new navigation macro - https://github.com/craftercms/craftercms/issues/4781
- Update with new dashboard widgets

### CODE REVIEW

File: templates/web/components/rich_text.ftl [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<#import "/templates/system/common/cstudio-support.ftl" as studio />
+<#import "/templates/system/common/crafter.ftl" as crafter />
 
-<div <@studio.componentAttr component=contentModel ice=true />>
-    ${contentModel.content_html!""}
-</div>
+<@crafter.tag $model=contentModel $field="content_html">
```
Comment (rart) [inline templates/web/components/rich_text.ftl:3]: use `@crafter.div`

File: templates/web/components/responsive_columns.ftl [hunk 2]
```diff
@@ -1,9 +1,12 @@
-<div class="row mb-5">
+<#import "/templates/system/common/crafter.ftl" as crafter />
+
+<@crafter.tag $field="columns_o" class="row mb-5">
```
Comment (rart) [inline templates/web/components/responsive_columns.ftl:3]: use `@crafter.div`

File: templates/web/components/responsive_columns.ftl [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
+  <#if contentModel.columns_o?? && contentModel.columns_o.item??>
     <#list contentModel.columns_o.item as column>
-      <div class="col-md-${column.columnSize_s} mb-4">
-          <#list column.content_o.item as component>
-              <@renderComponent component=component />
-          </#list>
-      </div>
+      <#assign index = column?index>
+      <@crafter.tag $field="columns_o" class="col-md-${column.columnSize_s} mb-4" $index=index>
```
Comment (rart) [inline templates/web/components/responsive_columns.ftl:7]: use `@crafter.div`

File: templates/web/category.ftl [hunk 4]
```diff
@@ -52,81 +55,83 @@
...
+                      Categories:
+                    <#else>
+                      Tags:
+                    </#if>
                 </h2>
               </div>
 
               <div class="col-md-12 col-lg-8">
-                <div class="row" <@studio.componentAttr component=taxonomy ice=true />>
+                <@crafter.tag $model=taxonomy class="row">
```
Comment (rart) [inline templates/web/category.ftl:87]: use `@crafter.div`

File: templates/web/category.ftl [hunk 5]
```diff
@@ -29,7 +31,7 @@
                   <div class="col-md-12">
                     <#list postsInfo.paginatedPosts as post>
                       <#assign postItem = siteItemService.getSiteItem(post.localId) />
-                      <div class="post-entry-horizontal" <@studio.componentAttr component=postItem ice=true />>
+                      <@crafter.tag $model=postItem class="post-entry-horizontal">
```
Comment (rart) [inline templates/web/category.ftl:34]: use `@crafter.div`

File: templates/web/about.ftl [hunk 6]
```diff
@@ -1,59 +1,60 @@
...
                 <div class="col-md-12">
-                  <#list postsInfo.paginatedPosts as post>
-                    <#assign postItem = siteItemService.getSiteItem(post.localId) />
-                    <div class="post-entry-horizontal" <@studio.componentAttr component=postItem ice=true />>
-                      <#assign url = postItem.storeUrl
-                        ?replace("/site/components", "")
-                        ?replace(".xml", "")
+                    <#list postsInfo.paginatedPosts as post>
+                      <#assign postItem = siteItemService.getSiteItem(post.localId) />
+                      <@crafter.tag $model=postItem class="post-entry-horizontal">
```
Comment (rart) [inline templates/web/about.ftl:37]: use `@crafter.div`

**Review by rart — CHANGES_REQUESTED** [review]
- Why is there a react build on the freemarker only version?
- For tags covered by our tag macros, make sure to use the specific tag instead of use `@crafter.tag` across the board. I tagged various instances, but do a global find making sure use `@crafter.tag` is only used for tags not covered by our macros. And, in fact if you find a tag we aren't covering, we should add it.
- There's a number of changes that are odd for the small update we're supposed to be doing. Please double check and double check the functional status of the entire blueprint (i.e. all pages work and are editable on page builder, forms look good, drag & drop works, etc)

**Comment by jvega190** [comment]
Closing this PR. The updates are done in here https://github.com/craftercms/wordify-blueprint/pull/37

**Pull request closed without merging**